lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -12,7 +12,7 @@ class HttpClient
     }
 
     def initialize(options={})
-      @logger = Cabin::Channel.get
+      @logger = options[:logger]
```

Comment:
are you sure? I only see it being added to Stud::Buffer's `@buffer_config` variable.